superset-frontend/src/explore/components/controls/SelectControl.jsx

```diff
@@ -160,6 +164,17 @@ export default class SelectControl extends React.PureComponent {
     return filterOption({ data: option }, text);
   }
 
+  handleSelectAll() {
+    /**
+     * Function to handle select all button.
+     * Adds all options to the selected values.
+     */
```

Comment:
```suggestion
  /**
   * Function to handle select all button.
   * Adds all options to the selected values.
   */
  handleSelectAll() {
```

Nit: normally we put function doc string above the function name.